code/game/machinery/hologram.dm

```diff
@@ -406,14 +409,17 @@ For the other part of the code, check silicon say.dm. Particularly robot talk.*/
 
 /obj/machinery/holopad/proc/set_holo(mob/living/user, var/obj/effect/overlay/holo_pad_hologram/h)
 	masters[user] = h
+	holorays[user] = new /obj/effect/overlay/holoray(loc)
 	var/mob/living/silicon/ai/AI = user
 	if(istype(AI))
 		AI.current = src
 	SetLightsAndPower()
+	move_hologram(user, loc) //Just so the holoray updates

```

Comment:
Or you could move the behavior you're trying to activate into a new proc. (This isn't a suggestion)

Reply:
You didn't have to be so passive-aggressive about it